### PR TITLE
Add unit tests for generate_apitoken.py and config.py

### PR DESCRIPTION
**Pull request by mushtaqak**
This PR adds unit tests for generate_apitoken.py and config.py

[EDUCATOR-880](https://openedx.atlassian.net/browse/EDUCATOR-880)

### CODE REVIEW

**Comment by mushtaqak** [comment]
@muzaffaryousaf @yro can you review this?

**Comment by mushtaqak** [comment]
Coverage shows **100%** for `generate_apitoken.py` and `config.py` :)

```
video_worker/__init__.py                                              157    102    35%
video_worker/abstractions.py                                          127     85    33%
video_worker/api_communicate.py                                       102      3    97%
video_worker/celeryapp.py                                              37     16    57%
video_worker/config.py                                                 46      0   100%
video_worker/generate_apitoken.py                                      34      0   100%
video_worker/generate_delivery.py                                      86     64    26%
video_worker/generate_encode.py                                       152    133    13%
video_worker/global_vars.py                                            16      0   100%
video_worker/reporting.py                                              77     58    25%
video_worker/validate.py                                               84     70    17%
video_worker/video_images.py                                           81     25    69%
```

File: video_worker/tests/test_generate_token.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        Tests `val_tokengen` method logs correct error message.
+        """
+        mock_post.return_value = Mock(status_code=400, text='', content={})
+        response = val_tokengen()
+        self.assertFalse(response)
+        mock_logger.error.assert_called_with('Token Gen Fail: VAL - Check VAL Config')
+
+    @patch('video_worker.generate_apitoken.requests.post')
+    @patch('video_worker.generate_apitoken.logger')
+    def test_val_tokengen(self, mock_logger, mock_post):
```
Comment (muzaffaryousaf) [inline video_worker/tests/test_generate_token.py]: mock_logger is not being used.

File: video_worker/tests/test_generate_token.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            Mock(status_code=200, text=response_data, content=response_data),
+            Mock(status_code=400, text='', content={})
+        ]
+        response = veda_tokengen()
+        self.assertFalse(response)
+        mock_logger.error.assert_called_with('VEDA Token Authorization')
+
+    @patch('video_worker.generate_apitoken.requests.post')
+    @patch('video_worker.generate_apitoken.logger')
+    def test_veda_tokengen(self, mock_logger, mock_post):
```
Comment (muzaffaryousaf) [inline video_worker/tests/test_generate_token.py]: mock_logger is not being used.

File: video_worker/config.py [hunk 3]
```diff
@@ -97,15 +93,3 @@ def _CONFIGURE(self):
...
-def main():
-    """
-    For example
-    """
-    V = WorkerSetup()
-    V.run()
-
-
-if __name__ == '__main__':
-    sys.exit(main())
```
Comment (yro) [inline video_worker/config.py]: @muzaffaryousaf Nope! They can all be removed.

**Comment by mushtaqak** [comment]
@muzaffaryousaf addressed feedback

**Comment by muzaffaryousaf** [comment]
@yro Need your feedback on this. Thanks.

**Pull request merged**